<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjouterGrapheDialog</class>
 <widget class="QDialog" name="AjouterGrapheDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNomGraphe">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom du graphe :</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkOriente">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Orienté</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkValue">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Valué</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonChargerDepuisFichier">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Charger depuis un fichier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSaisieGraphique">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Saisie graphique</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AjouterGrapheDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AjouterGrapheDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
